doc/building.md

```diff
@@ -57,6 +57,7 @@ Dependency list:
     CR    qt6 >=6.2 (Core, Quick, QuickControls, Multimedia modules)
     CR    toml11
     CR  O vulkan
+       S  clang-tidy
```

Comment:
please keep the alphabetical sorting :)

Reply:
Hi @TheJJ , I am actually a little confused. Do you mean alphabetical sorting with respect to "S"?

Reply:
Or I guess it's right alphabetical sorting within this:
 R    dejavu font
    CR    eigen >=3
    CR    freetype2
    CR    fontconfig
    CR    harfbuzz
    C   O include-what-you-use
    CR    nyan  (https://github.com/SFTtech/nyan)
    CR  O ncurses
    C     mako
    CR    opusfile
    CRA   opus
    CRA   ogg
       S  pycodestyle
    C     pygments
       S  pylint
    CR    qt6 >=6.2 (Core, Quick, QuickControls, Multimedia modules)
    CR    toml11
    CR  O vulkan